controllers/static/static_controller.go

```diff
@@ -0,0 +1,99 @@
+package static
+
+import (
+	"context"
+	"time"
+
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+
+	flowslatest "github.com/netobserv/network-observability-operator/apis/flowcollector/v1beta2"
+	"github.com/netobserv/network-observability-operator/controllers/consoleplugin"
+	"github.com/netobserv/network-observability-operator/controllers/reconcilers"
+	"github.com/netobserv/network-observability-operator/pkg/helper"
+	"github.com/netobserv/network-observability-operator/pkg/manager"
+	"github.com/netobserv/network-observability-operator/pkg/manager/status"
+)
+
+const (
+	initReconcileAttempts = 5
+)
+
+type Reconciler struct {
+	client.Client
+	mgr    *manager.Manager
+	status status.Instance
+}
+
+func Start(ctx context.Context, mgr *manager.Manager) error {
+	log := log.FromContext(ctx)
+	log.Info("Starting Static controller")
+	r := Reconciler{
+		Client: mgr.Client,
+		mgr:    mgr,
+		status: mgr.Status.ForComponent(status.StaticPlugin),
+	}
+
+	// force reconcile at startup
+	go r.InitReconcile(ctx)
+
+	return ctrl.NewControllerManagedBy(mgr).
+		For(&flowslatest.FlowCollector{}, reconcilers.IgnoreStatusChange).
+		Named("staticPlugin").
+		Complete(&r)
+}
+
+func (r *Reconciler) InitReconcile(ctx context.Context) {
+	log := log.FromContext(ctx)
+	log.Info("Initializing resources...")
+
+	for attempt := range initReconcileAttempts {
+		// delay the reconcile calls to let some time to the cache to load
+		time.Sleep(5 * time.Second)
+		_, err := r.Reconcile(ctx, ctrl.Request{})
+		if err != nil {
+			log.Error(err, "Error while doing initial reconcile", "attempt", attempt)
+		} else {
+			return
+		}
+	}
+}
```

Comment:
@OlivierCazade I modified the PR to be in a dedicated controller.

As you can see, I force the `Reconcile` call during the `Start` function so OLM can't interpret the result containing Requeue / RequeueAfter here.

Since we don't create a dedicated CR for static plugin, I don't think we can rely on these here.

WDYT ?